modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc

```diff
@@ -0,0 +1,280 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configure-rollout-route-traffic-using-openshift-routes_{context}"]
+= Configuring Argo Rollouts to route traffic by using OpenShift Routes
+
+You can use OpenShift Routes to configure Argo Rollouts to create routes, rollouts, and services.
+
+The following example procedure creates a route, a rollout, and two services. It then gradually routes an increasing percentage of traffic to a canary version of the application before that canary state is marked as successful and becomes the new stable version.
+
+.Prerequisites
+
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed the {gitops-title} on your {OCP} cluster.
+* You have installed Argo Rollouts on your {OCP} cluster. For more information, see xref:../argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc#gitops-creating-rolloutmanager-custom-resource_using-argo-rollouts-for-progressive-deployment-delivery[Creating a RolloutManager custom resource].
```

Comment:
Per the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references) you can't put xrefs in modules. If you want a link to this page, you'll have to add it in an additional resources section underneath where the module is included in the assembly. Here's some guidance (see the parts about using it on the module level): https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

As for the text in this module, you have to take out the xref and replace it with some formatting like quotation marks:
```suggestion
* You have installed Argo Rollouts on your {OCP} cluster. For more information, see "Creating a RolloutManager custom resource".
```

Reply:
@varshab1210 - I would like to revert your earlier feedback and remove the links from prereqs. Adding quotation marks and since the links are already included in the Additional resources section, removing the links would be the appropriate course of action to fix this comment.